app/assets/javascripts/application.js

```diff
@@ -21,5 +21,5 @@
 $(function(){ $(document).foundation(); });
 
 $(document).ajaxError(function(event, xhr, settings, exception) {
-  return alert("Sorry, the form couldn't be saved. YOU SUCK. I'm sorry, I didn't have my coffee.");
+  return alert(exception));
```

Comment:
Missing semicolon.<br>Expected an identifier and instead saw ')'.<br>Expected an assignment or function call and instead saw an expression.<br>'alert' is not defined.